packages/cronjob-rss-feed/configs.js

```diff
@@ -0,0 +1,14 @@
+export const config = {
+  apiUrl: 'https://kids-cms.twreporter.org/api/graphql',
+  baseUrl: 'https://kids.twreporter.org/',
+  bucketName: 'dev-kids-storage.twreporter.org',
```

Comment:
use `kids-storage.twreporter.org` bucket?

Reply:
This code is outdated